src/utils.ts

```diff
@@ -355,6 +350,17 @@ export function extractInt(str: string): number {
 	return parseInt(str.match(/\d+/)![0]);
 }
 
+export function getLargestFile(files: File[]): File {
+	return files.reduce((a, b) => (a.length > b.length ? a : b));
+}
+
+export function getNewestFileAge(files: File[]): number {
+	return files.reduce(
+		(acc, file) => Math.max(acc, statSync(file.path).mtimeMs),
```

Comment:
In a tight loop like this, I prefer to do `fs` calls async:

```ts
import { stat } from 'fs/promises';

async function getNewestFileAge(files: File[]): Promise<number> {
	return Math.max(...await Promise.all(files.map(f => stat(f.path)));
}
```

also, these functions are a little bit more business logicky than most utils. It might make more sense for them to exist in `searchee.ts`, what do you think?